### DISCUSSION TITLE
Only some videos are uploaded to minio within the same run

### DISCUSSION THREAD

**Discussion by TheZoker** [sorry-cypress/sorry-cypress#783] [Q&A] [answered]
Hi guys,

I'm trying to use sorry-cypress together with minio as a storage solution.
Now when our test runs on bamboo, it uploads videos to the minio storage.
But only for some test suites. For some the browser tries to load the video, but the file is not there in the minio storage.
So within the same run, some videos get uploaded, some not.

Any idea what is going on and how I can debug/fix this?
Thanks!

**Comment by agoldis** [comment]
Try following this guide: https://currents.dev/readme/integration-with-cypress/troubleshooting
and correlate the debug messages with the backend services logs

**Reply by TheZoker** [reply]
Thanks for the hint! Really appreciate it!

I run the job with the debug env and this is what I found:
```
error	20-May-2023 20:18:20	docker-artemis-cypress-1  | 2023-05-20T20:18:20.740Z currents:api network request: { method: 'put', url: '<secret-url>/sorry-cypress/5a0efe75-0027-43eb-b382-211a706251d7.mp4?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=xxx%2F20230520%2Fus-east-1%2Fs3%2Faws4_request&X-Amz-Date=20230520T201820Z&X-Amz-Expires=604800&X-Amz-SignedHeaders=host&X-Amz-Signature=xxx', headers: { Accept: 'application/json, text/plain, */*', 'Content-Type': 'video/mp4', 'x-cypress-request-attempt': 0, 'x-cypress-version': '12.11.0', 'x-ccy-version': '1.7.4', 'x-cypress-run-id': '174d8ee97ab0f232fc275ca8fb7ec97e' }, data: 'buffer' }
error	20-May-2023 20:18:20	docker-artemis-cypress-1  | 2023-05-20T20:18:20.778Z currents:artifacts failed uploading video /app/artemis/src/test/cypress/videos/ExamAssessment.cy.ts.mp4. Error: Error: Request failed with status code 413 at /app/artemis/src/test/cypress/node_modules/lib/httpClient/httpClient.ts:106:13 at processTicksAndRejections (node:internal/process/task_queues:95:5) at uploadFile (/app/artemis/src/test/cypress/node_modules/lib/upload.ts:19:3) at /app/artemis/src/test/cypress/node_modules/lib/lang.ts:16:17 at uploadArtifacts (/app/artemis/src/test/cypress/node_modules/lib/artifacts.ts:37:5) at async Promise.all (index 0)
```

Is there any way to debug this further? Why is the response code 413 here? Is the video file to big?

**Reply by TheZoker** [reply, marked as answer]
Update: I found out, that nginx only allows 1MB of body size. I increased this by setting `client_max_body_size 100M;` within the `nginx.conf` file
